@navikt/core/css/darkside/date.darkside.css

```diff
@@ -1,222 +1,206 @@
+    padding: 0;
+  }
+
+  .navds-date__modal-body > & {
+    padding: 0;
   }
 }
 
-.navds-date__year-label {
+.navds-date__caption {
   display: flex;
+  justify-content: space-between;
   align-items: center;
+  gap: var(--ax-spacing-1);
+  padding-inline: var(--ax-spacing-1);
 }
 
-.navds-date__wrapper,
-.navds-date__standalone-wrapper {
-  height: fit-content;
-  width: fit-content;
-}
+.navds-date__caption-button {
+  width: 2rem;
+  height: 2rem;
 
-.navds-date .navds-date__field {
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-  position: relative;
+  /* TODO: ?? */
+  &,
+  &:disabled,
+  &:hover {
+    color: var(--ax-text-default);
+  }
 }
 
-.navds-date__field-wrapper {
-  display: inline-flex;
-  align-items: center;
-  position: relative;
-  width: fit-content;
+.navds-date__caption__month .navds-select__container select {
+  text-transform: capitalize;
 }
 
-.navds-date .rdp-day_selected,
-.navds-monthpicker__month--selected {
-  color: var(--ac-date-selected-text, var(--a-text-on-action));
-  background: var(--ac-date-selected-bg, var(--a-surface-action-selected));
-  cursor: pointer;
+.navds-date__caption__year {
+  width: 5rem;
+  border: 1px solid var(--ax-border-default);
+  border-radius: var(--ax-border-radius-medium);
 }
 
-.navds-date .rdp-day_disabled {
-  cursor: not-allowed;
-  text-decoration: line-through;
-  background-color: var(--ac-date-disabled-bg, var(--a-surface-neutral-subtle));
-  color: var(--ac-date-disabled-text, var(--a-text-subtle));
+.navds-date__caption__month {
+  border: 1px solid var(--ax-border-default);
+  border-radius: var(--ax-border-radius-medium);
 }
 
-.navds-date .rdp-button:where(:not(.rdp-day_selected, [disabled])):hover,
-.navds-date__month-button:where(:not(.rdp-day_selected, [disabled])):hover {
-  background: var(--ac-date-hover-bg, var(--a-surface-action-subtle-hover));
+/* Monthpicker */
+.navds-date__month-button {
+  all: unset;
+  text-align: center;
+  width: 3rem;
+  height: 3rem;
+  text-transform: capitalize;
+  border-radius: var(--ax-border-radius-medium);
   cursor: pointer;
 }
 
-.navds-date .rdp-day_today {
-  box-shadow: 0 0 0 1px var(--ac-date-today-border, var(--a-border-action-selected));
-  text-decoration: underline;
-}
-
-.navds-date__caption {
+.navds-date__year-label {
   display: flex;
-  justify-content: space-between;
   align-items: center;
-  gap: var(--a-spacing-1);
-  padding-inline: var(--a-spacing-1);
 }
 
-.navds-date__caption-button,
-.navds-date__caption-button:disabled,
-.navds-date__caption-button:disabled:hover {
-  color: var(--ac-date-caption-text, var(--a-text-default));
-}
-
-.navds-date__field-input {
-  padding-right: var(--a-spacing-12);
-}
+.navds-date__wrapper,
+.navds-date__standalone-wrapper {
+  height: fit-content;
+  width: fit-content;
 
-.navds-form-field--small .navds-date__field-input {
-  padding-right: var(--a-spacing-8);
+  button {
+    &:focus-within {
+      outline: 2px solid var(--ax-border-focus);
```

Comment:
```suggestion
      outline: 2px solid var(--ax-border-focus);
      outline-offset: -1px;
```

Reply:
I think the `button` selector is to wildcardy since it targets native buttons used in caption to

Reply:
This target the monthpicker buttons differently than datepicker
![Screenshot 2024-11-25 at 12 55 41](https://github.com/user-attachments/assets/b56b7b61-8bcd-44fb-ba22-0cf4548b81ad)